src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -693,37 +695,41 @@ and group files. To activate this behavior, set `pa.ldap.authentication.fallback
 
 === PAM
 
-The ProActive Scheduler & Resource Manager are able to interact with *PAM (Linux Pluggable Authentication Modules)* to check users login/password.
+The ProActive Scheduler & Resource Manager are able to interact with Linux *PAM (Pluggable Authentication Modules)* to check users login/password.
 It is not currently possible to retrieve linux system group memberships for users. Thus groups must be managed using the `PROACTIVE_HOME/config/authentication/group.cfg` file.
 
-The *first step* to authenticate users with PAM is to configure the PAM login methods in the scheduler and rm settings:
-
-
+1. The *first step* to authenticate users with PAM is to configure the PAM login methods in the Scheduler and RM settings:
++
 .PROACTIVE_HOME/config/rm/settings.ini
 ----
 pa.rm.authentication.loginMethod=RMPAMLoginMethod
 ----
-
++
 .PROACTIVE_HOME/config/scheduler/settings.ini
 ----
 pa.scheduler.core.authentication.loginMethod=SchedulerPAMLoginMethod
 ----
++
 
-The *second step* is to copy the file `PROACTIVE_HOME/config/authentication/proactive-jpam` to `/etc/pam.d` (you must be root to perform this operation).
-
+2. The *second step* is to copy the file `PROACTIVE_HOME/config/authentication/proactive-jpam` to `/etc/pam.d` (you must be root to perform this operation).
```

Comment:
Since you are now using an enumeration list it is no longer required to name steps. Sentences could be rephrased.